### DISCUSSION TITLE
Allow binding right and left modifies separately

### DISCUSSION THREAD

**Discussion by casey** [rxhanson/Rectangle#674] [Ideas]
Thanks for Rectangle, it's great!

I don't use the right modifiers on my keyboard, so I like to bind them to global shortcuts. Because I don't use them for anything else, I can use very short bindings, like `right command + left arrow` to move a window to the left. It would be great if I could bind rectangle shortcuts to modifier keys on the right side. I actually suspect not using modifiers on the right-hand side is pretty common, so other users might like this too. The feature could take the form of a checkbox, which, when checked, records bindings that are side-specific.

**Comment by rxhanson** [comment]
Thanks for the idea. I've thought about this one before. From a developer perspective, it's not that appealing for Rectangle because it would require keystroke listening, where Rectangle currently registers hotkeys and does not have to do any keystroke listening. I don't have plans to add it into Rectangle myself, but would consider a pull request for it. I will keep it in mind for some of my other apps, though.
